pkg/cmd/label/http.go

```diff
@@ -1,14 +1,13 @@
-package list
+package label
 
 import (
 	"net/http"
 
 	"github.com/cli/cli/v2/api"
 	"github.com/cli/cli/v2/internal/ghrepo"
-	"github.com/cli/cli/v2/pkg/cmd/label/shared"
 )
 
-func listLabels(client *http.Client, repo ghrepo.Interface, limit int) ([]shared.Label, int, error) {
+func listLabels(client *http.Client, repo ghrepo.Interface, limit int) ([]label, int, error) {
```

Comment:
Is there a good reason not to just move `listLabels` and `createLabels` to the already existing `shared` package? I think that path conforms with what we have done with other commands but am open to hearing reasons why this is a might be a better solution.

Reply:
I don't have a strong preference and I think it is reasonable to keep as much non-exported as possible.